src/pages/storage/StoragePools.tsx

```diff
@@ -108,24 +151,25 @@ const StoragePools: FC = () => {
       <Row>
         {hasStoragePools && (
           <>
-            <div className="u-align--right">
-              <AddStorageBtn project={project} />
+            <div className="upper-controls-bar">
+              <CreateStoragePoolBtn project={project} />
             </div>
-            <MainTable
-              headers={headers}
-              rows={rows}
-              paginate={30}
-              responsive
-              sortable
-              className="u-table-layout--auto"
-              emptyStateMsg={
-                isLoading ? (
-                  <Loader text="Loading storage pools..." />
-                ) : (
-                  "No data to display"
-                )
-              }
-            />
+            <ScrollableTable dependencies={[pools]}>
+              <MainTable
+                headers={headers}
+                rows={rows}
+                paginate={30}
+                sortable
+                className="u-table-layout--auto storage-pool-table"
+                emptyStateMsg={
+                  isLoading ? (
+                    <Loader text="Loading storage pools..." />
+                  ) : (
+                    "No data to display"
+                  )
+                }
+              />
+            </ScrollableTable>
```

Comment:
Adding the `ScrollableTable` is a step forward, but it still feels like something is missing. Switching tabs in the "Storage" page alone, I notice these discrepancies:

1. The absence of the search/search and filter.
2. The absence of pagination.

Would it be good to add these? The "Pools" tab looks "outdated" and inconsistent with the other pages/tabs otherwise.

Reply:
We don't expect more than a handful of pools, hence the tab is not paginated or filterable to make it lighter.

Reply:
Still doesn't seem right to have it everywhere else except here. @piperdeck what do you think about this?

Reply:
We can also treat it as a separate improvement. I see that other pages are inconsistent too: Profiles still have the pagination at the bottom, Networks don't have anything at all - not even the `ScrollableTable`... There's a lot of work to do to reach consistency :)